### PR TITLE
Fix/day summary and payrun

### PR DESCRIPTION
**Pull request by mugnivenko**

### CODE REVIEW

File: components/HomeCare/DaySummary.js [hunk 1]
```diff
@@ -86,8 +86,8 @@ const DaySummary = ({ daySummaryItem, edit = (item) => console.log(item), remove
               <div className="column">
                 <span>Need Addressing</span>
                 <p>
-                  {formattedNeedAddressing}{' '}
-                  {formattedNeedAddressing.length !== careSummary.needAddressing.length && (addressTextCollapsed ? (
+                  {`${formattedNeedAddressing} `}
```
Comment (akeamwoodsND) [inline components/HomeCare/DaySummary.js:89]: {formattedNeedAddressing}

File: components/HomeCare/DaySummary.js [hunk 2]
```diff
@@ -153,7 +153,7 @@ const DaySummary = ({ daySummaryItem, edit = (item) => console.log(item), remove
                 <span>What should be done</span>
                 <p>
                   {whatShouldBeDone}{' '}
```
Comment (akeamwoodsND) [inline components/HomeCare/DaySummary.js:155]: remove {' '}

File: components/PayRuns/PayRunCollapsedContent.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
-import { getNumberWithCommas } from '../../service/helpers';
+import PayRunInvoiceItems from './PayRunInvoiceItems'
 
 const PayRunCollapsedContent = ({ invoice = {} }) => {
   const { invoiceItems } = invoice;
-  let totalToPay = 0;
   if (!invoiceItems) return React.Fragment;
+
+  const invoiceClaimedByHackney = invoiceItems.filter(item => item.claimedBy === 'Hackney');
+  const invoiceClaimedByOther = invoiceItems.filter(item => item.claimedBy !== 'Hackney');
```
Comment (akeamwoodsND) [inline components/PayRuns/PayRunCollapsedContent.js:9]: we don't need this variable, this variable is the same as:     !invoiceClaimedByHackney

File: components/PayRuns/PayRunInvoiceItems.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { currency } from '../../constants/strings';
```
Comment (akeamwoodsND) [inline components/PayRuns/PayRunInvoiceItems.js:2]: 'constants/...'

File: components/PayRuns/PayRunInvoiceItems.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+import { getNumberWithCommas } from '../../service/helpers';
```
Comment (akeamwoodsND) [inline components/PayRuns/PayRunInvoiceItems.js:3]: 'service/......'

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]

File: components/HomeCare/DaySummary.js (hunk 1 above)
Comment (mugnivenko) [reply components/HomeCare/DaySummary.js:89]: we need space here

File: components/HomeCare/DaySummary.js (hunk 2 above)
Comment (mugnivenko) [reply components/HomeCare/DaySummary.js:155]: replace similar code on {`${variable} `} where we need space

**Review by akeamwoodsND — APPROVED** [review]
Looks good!

**Pull request merged**